docs/docs/benchmark.md

```diff
@@ -1,96 +1,262 @@
-# Benchmarking
+# Benchmarking suites
 
-<p>OpenML significantly simplifies benchmarking by:</p>
-</li><li>offering a large set of datasets in standardized data formats
-</li><li>easily download through a wide range of [APIs](APIs) and existing client libraries</a>
-</li><li>machine-readable meta-information on all the datasets
-</li><li>sharing of benchmarking results in a reproducible way through the [APIs](APIs), enabling large scale comparisons
-</li></ul>
+OpenML offers <b>benchmarking suites</b>: curated, comprehensive sets of machine learning datasets,
+covering a wide spectrum of domains and statistical properties. This makes benchmarking results more comparable,
+more interpretable, and allows more standardized analysis of algorithms under different conditions.
 
-In addition, OpenML offers <b>benchmarking suites</b>: curated, comprehensive sets of machine learning datasets,
-covering a wide spectrum of domains and statistical properties and standardized evaluation procedures (defined in tasks).
-This makes benchmarking results more comparable, more complete, and allows more standardized analysis of algorithms under different conditions.
+Benchmarking suites make benchmarking a whole lot easier:
+- all datasets are uniformly formatted in standardized data formats
+- they can be easily downloaded programmatically through [APIs and client libraries](APIs)
+- they come with machine-readable [meta-information](https://www.openml.org/search?type=measure&q=+measure_type%3Adata_quality), such as the occurrence of missing values, to train algorithms correctly
+- standardized train-test splits are provided to make results more comparable
+- results can be shared in a reproducible way through the [APIs](APIs)
+- results from other users can be easily downloaded and reused
 
+## Terminology: Studies and Benchmark Suites
+OpenML offers the ability to bundle several objects on the server together on a
+single page. With a slight overload of terminology, these are called `studies'.
+OpenML supports two types of studies:
+- benchmark suites: collection of tasks. OpenML automatically organizes all 
+tasks and the correpsonding datasets on a single server page. These are the main
```

Comment:
Not sure if this will render correctly or whether you need to indent by two spaces (to make the sentence start at the same level of indentation).